lib/package.gi

```diff
@@ -1190,30 +1189,27 @@ InstallGlobalFunction( DefaultPackageBannerString, function( inforec )
 #F  DirectoriesPackagePrograms( <name> )
 ##
 InstallGlobalFunction( DirectoriesPackagePrograms, function( name )
-    local arch, dirs, info, version, r, path;
+    local info, ppath;
 
-    arch := GAPInfo.Architecture;
-    dirs := [];
     # We are not allowed to call
     # `InstalledPackageVersion', `TestPackageAvailability' etc.
     info:= PackageInfo( name );
     if IsBound( GAPInfo.PackagesLoaded.( name ) ) then
       # The package is already loaded.
-      version:= GAPInfo.PackagesLoaded.( name )[2];
+      ppath:= GAPInfo.PackagesLoaded.( name )[1];
```

Comment:
What does ppath stand for? "package path"? I first thought it was a typo. So, optional suggestion: revert to `path`, or use a more expressive name, like `pkg_path`?

Reply:
I wanted the code of `DirectoriesPackageLibrary` and `DirectoriesPackagePrograms` to be similar, and `path` was already used in one of them.
I can choose another variable name, but I think this discussion is not useful.